### PR TITLE
Implemented Json data for blog

### PR DESCRIPTION
**Pull request by Sowmiya89**
merge request

### CODE REVIEW

File: autozone-poc/src/components/Blog/Blog.js [hunk 1]
```diff
@@ -1,239 +1,87 @@
...
+
+  return (
+    <div>
+      <div
+        data-testid="background"
+        className="blog-background-image"
+        style={{
+          backgroundImage:
+            "url(" +
+            "https://www.autozone.com/diy/wp-content/uploads/2020/04/is-my-car-battery-dead-1024x683.jpg" +
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/Blog/Blog.js]: Why it is hard coded

File: autozone-poc/src/components/Blog/Blog.js [hunk 2]
```diff
@@ -1,239 +1,87 @@
...
-                      imgpath: blog1
-                    }
-                    }}>
-                    Continue reading <i className="fa fa-angle-right"></i>
-                    </NavLink>
-                </div>
-              </div>
-            </div>
+import { BrowserRouter as Router,NavLink } from 'react-router-dom';
+import list from "../Blog/blog.json";
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/Blog/Blog.js]: kindly use axios call or fetch call to implement to use the article in the page

**Review by gokulwebdeveloper — CHANGES_REQUESTED** [review]

File: autozone-poc/src/components/Blog/Blog.js (hunk 1 above)
Comment (Sowmiya89) [reply autozone-poc/src/components/Blog/Blog.js]: Fixed

File: autozone-poc/src/components/Blog/Blog.js (hunk 2 above)
Comment (Sowmiya89) [reply autozone-poc/src/components/Blog/Blog.js]: Added axios call

**Review by gokulwebdeveloper — APPROVED** [review]

**Comment by gokulwebdeveloper** [comment]
Please resolve conflict and let me know so that I will merge

**Pull request merged**